yearzero/YZRoll.js

```diff
@@ -560,7 +562,7 @@ class YZRoll {
 		else {
 			out.push('=', this.sum());
 		}
-		if (this.pushed) out.unshift('(pushed)');
+		if (this.pushed) out.unshift(`(${__('pushed', this.lang)})`);
```

Comment:
IIRC, this is only used in logs, so no need to translate it.

Reply:
Seems to be right, i reverted it in [this commit](https://github.com/Stefouch/sebedius-yearzero-discord-bot/pull/106/commits/02d2235bc4dbd46f6f5286f449825c531fc1422b).